widgetry/src/style/button_style.rs

```diff
@@ -106,15 +106,14 @@ impl<'a, 'c> Style {
     pub fn btn_popup_icon_text(&self, icon_path: &'a str, text: &'a str) -> ButtonBuilder<'a, 'c> {
         // The text is styled like an "outline" button, while the image is styled like a "solid"
         // button.
-        let solid_style = &self.btn_tab;
         self.btn_outline
             .btn()
             .label_text(text)
             .image_path(icon_path)
             .image_dims(25.0)
-            .image_color(solid_style.fg, ControlState::Default)
-            .image_bg_color(solid_style.bg, ControlState::Default)
-            .image_bg_color(solid_style.bg_hover, ControlState::Hovered)
+            .image_color(self.btn_solid.fg, ControlState::Default)
```

Comment:
perviously we used `self.btn_tab` to style things like the icon portion of the "pop up" buttons (e.g. "switch scenario"), but the new tab styling isn't compatible with that usage, so I've introduced another button_style.